package.json

```diff
@@ -1035,6 +1035,17 @@
                         "type": "boolean",
                         "description": "%azureFunctions.endOfLifeWarning%",
                         "default": true
+                    },
+                    "azureFunctions.functionSubpath": {
+                        "type": "string",
+                        "default": "src/functions",
+                        "description": "%azureFunctions.functionSubpath%"
+                    },
+                    "azureFunctions.showNodeProgrammingModel": {
```

Comment:
Should this be done as an experiment instead?

Reply:
Similar to the Python v2 model, we're using the flag because the local experience totally works, but the Azure rollout isn't complete yet. Once that happens, we can remove the flag, but we wanted to give users/internals a chance to use the local experience and offer feedback.

Reply:
The same can be accomplished with a setting-based experiment, though. Users _can_ override the experimental setting that they are assigned, FWIW.

Reply:
Fair enough! And then we could remove it without a release. I'm not sure how much effort it really is to pilot an experiment, but if you think it's worth it, let's do it.

I'll definitely need some assistance with that though.

Reply:
I guess the best way to answer that is, do we want more or less people to see the feature? If we go with off-by-default and no experiment, then very few folks will discover it. An experiment could greatly increase the number of folks that see the feature.

Reply:
Yeah. And naturally, you can control not just the exposure, but _when_ you turn it on. So we could leave it all hidden (including the setting) until we're ready. You can also turn it off if things go sideways.